src/main/java/educationalproject/programmingstuff/servicies/CommodityItemServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package educationalproject.programmingstuff.servicies;
+
+import educationalproject.programmingstuff.model.CommodityItem;
+import educationalproject.programmingstuff.repositories.CommodityItemRepository;
+import educationalproject.programmingstuff.servicies.dto.CommodityItemResponseDto;
+import educationalproject.programmingstuff.servicies.mappers.CommodityItemResponseMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.annotation.Validated;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Validated
+public class CommodityItemServiceImpl implements CommodityItemService {
+
+    @Autowired
+    private CommodityItemRepository commodityItemRepository;
+
+    @Autowired
```

Comment:
annotations not needed, but make these fields final

Reply:
Looks like the same issue as "Order1" was. Tests didn't start without "Autowired" annotations. Now starts. Deleted.